diff --git a/node_modules/artillery/dist/lib/cmds/run.js b/node_modules/artillery/dist/lib/cmds/run.js
index cfd733b..ff2f6d6 100644
--- a/node_modules/artillery/dist/lib/cmds/run.js
+++ b/node_modules/artillery/dist/lib/cmds/run.js
@@ -9,7 +9,6 @@ var __rewriteRelativeImportExtension = (this && this.__rewriteRelativeImportExte
 import { createRequire } from 'node:module';
 import { promisify as p } from 'node:util';
 import { Args, Command, Flags } from '@oclif/core';
-import _csv from 'csv-parse';
 import createDebug from 'debug';
 import { CommonRunFlags } from "../cli/common-flags.js";
 const debug = createDebug('commands:run');
diff --git a/node_modules/artillery/dist/lib/util/prepare-test-execution-plan.js b/node_modules/artillery/dist/lib/util/prepare-test-execution-plan.js
index 52236a6..9649c0d 100644
--- a/node_modules/artillery/dist/lib/util/prepare-test-execution-plan.js
+++ b/node_modules/artillery/dist/lib/util/prepare-test-execution-plan.js
@@ -2,7 +2,7 @@ import fs from 'node:fs';
 import { createRequire } from 'node:module';
 import path from 'node:path';
 import { promisify as p } from 'node:util';
-import csv from 'csv-parse';
+import { parse as csv } from 'csv-parse';
 import createDebug from 'debug';
 const debug = createDebug('artillery');
 // Used for: loading user-provided .js scripts and esbuild-generated
diff --git a/node_modules/artillery/lib/cmds/run.ts b/node_modules/artillery/lib/cmds/run.ts
index 1d7eb6f..9a08625 100644
--- a/node_modules/artillery/lib/cmds/run.ts
+++ b/node_modules/artillery/lib/cmds/run.ts
@@ -1,7 +1,6 @@
 import { createRequire } from 'node:module';
 import { promisify as p } from 'node:util';
 import { Args, Command, Flags } from '@oclif/core';
-import _csv from 'csv-parse';
 import createDebug from 'debug';
 import { CommonRunFlags } from '../cli/common-flags.ts';
 
diff --git a/node_modules/artillery/lib/util/prepare-test-execution-plan.ts b/node_modules/artillery/lib/util/prepare-test-execution-plan.ts
index f8ea361..706c1b5 100644
--- a/node_modules/artillery/lib/util/prepare-test-execution-plan.ts
+++ b/node_modules/artillery/lib/util/prepare-test-execution-plan.ts
@@ -2,7 +2,7 @@ import fs from 'node:fs';
 import { createRequire } from 'node:module';
 import path from 'node:path';
 import { promisify as p } from 'node:util';
-import csv from 'csv-parse';
+import { parse as csv } from 'csv-parse';
 import createDebug from 'debug';
 
 const debug = createDebug('artillery');
